src/NUnitFramework/nunitlite.runner/AutoRun.cs

```diff
@@ -86,6 +86,10 @@ public int Execute(string[] args)
                 return TextRunner.OK;
             }
 
+            // We already showed version as a part of the header
+            if (options.ShowVersion)
+                return TextRunner.OK;
+
```

Comment:
I've already done the merge, although not in this branch. That's probably why I forgot about it. :-)

In any case, for this branch to stand on it's own, I'll fix that as well.